src/java.desktop/share/classes/java/awt/Component.java

```diff
@@ -4132,9 +4132,9 @@ private void updateInternalBuffers() {
         }
 
         /**
-         * Gets direct access to the back buffer, as an image.
+         * Provides direct access to the back buffer as an image.
          *
-         * @return direct access to the back buffer, as an image.
+         * @return direct access to the back buffer as an image.
```

Comment:
Hm… This doesn't look right.
```suggestion
         * @return the back buffer as an image.
```